### ISSUE TITLE
Set up an auto-refresh

### ISSUE THREAD

**Issue by dlucazeau** [mingo-app/mingo#398]
Hello
Some lists reflect logs. It would then be interesting to have an automatic refresh option.

Regards

**Comment by tothradoslav** [comment]
yes, this is in our roadmap. While at it, we could explore the best way to implement this. Just some quick ideas:

- list could be set to refresh every X seconds
- list could be set to watch changes and notify user when any of the docs have changed and thus the list is outdated
- thinking of watch, refresh every X seconds is not even necessary, as we know when it's time to refresh
- one option could be to watch for changes and mark documents that have changed in the meantime

Other ideas?

**Comment by dlucazeau** [comment]
Good ideas, 
this is the first point that corresponds best to my current use.

**Comment by Jeannot-Muller** [comment]
The solution with the less performance impact would be my favorite. I think the watch is best. Only update, when needed.

**Comment by tothradoslav** [comment]
In case of "watch", which I agree is the best solution, would the list be automatically updated or just notify the user that the list is outdated?

**Comment by Jeannot-Muller** [comment]
As mingo is a dev tool, I would argue that a notification is better. That will give you the possibility to first look at the "old" values, before you decide to do an active refresh. But you will probably get 10 answers if you ask 5 people ;-)

**Comment by tothradoslav** [comment]
watchers have been implemented in Mingo v2, please check it out. If you have issues with it, please submit a new issue.
